apps/etterlatte-hendelser-pdl/src/main/kotlin/hendelserpdl/Dodsmeldinger.kt

```diff
@@ -64,12 +63,12 @@ class DodsmeldingerRapid(private val context: RapidsConnection) : IDodsmeldinger
     val logger = LoggerFactory.getLogger(this.javaClass)
 
     override fun personErDod(ident: String, doedsdato: String?, endringstype: Endringstype) {
```

Comment:
Dette er død kode? not pun intended

Reply:
Meget mulig, jeg kjenner ikke til detaljene i appen. Jeg kan se om det er ubrukt og fjerne i så fall

Reply:
Du hadde helt rett, denne koden er død og slettes